test/build.sh

```diff
@@ -128,7 +128,7 @@ function build-pr() {
   export REGISTRY="${NPD_STAGING_REGISTRY}/pr/${PR}"
   export VERSION=$(get-version)
   export TAG="${VERSION}"
-  make push
+  make push-tar
```

Comment:
NOTE: `pull-npd-build` already runs `make build` which includes `build-container`, so there's no reason to build it in e2e runs.

Reply:
(specifically it runs `./test/build.sh install-lib && make build`)

Reply:
Can you add a comment here to explain that this is only used by presubmit e2e? The build job is using `make build` which covers container build already?